frontend/src/components/Burger.js

```diff
@@ -27,33 +19,36 @@ class Burger extends Component {
       });
     }
 
-    showSettings (event) {
-      event.preventDefault();
+    showSettings (e) {
+      e.preventDefault();
     }
 
-    logout(event) {
+    logout(e) {
       localStorage.clear();
```

Comment:
This should probably redirect.

Reply:
The log out is a Link object, it returns to the login page. Do you still want to redirect from this function instead?